docs/deployment/kubernetes.md

```diff
@@ -4,22 +4,22 @@ This guide is for deployment to a vanilla Kubernetes 1.8 or 1.9 cluster running
 
 ## Kubernetes
 
-OpenFaaS is Kubernetes-native and uses *Deployments*, *Services* and *Secrets*. For more detail check out the ["faas-netes" repository](https://github.com/openfaas/faas-netes).
+OpenFaaS is Kubernetes-native and uses _Deployments_, _Services_ and _Secrets_. For more detail check out the ["faas-netes" repository](https://github.com/openfaas/faas-netes).
```

Comment:
Any reason for changing this?

Reply:
I think a linter changed that and I didn't realize.  For some reason, there is a linter in vscode that I don't now how to disable